lib/middleware/iriTemplate.js

```diff
@@ -19,17 +19,22 @@ function createTermFromVariable ({ template, value }) {
     return value
   }
 
-  const matches = value.match(literalValueRegex)
-  if (matches) {
-    let datatypeOrLanguage = matches.groups.language
-    if (matches.groups.datatype) {
-      datatypeOrLanguage = rdf.namedNode(matches.groups.datatype)
+  const parseValue = value => {
```

Comment:
```suggestion
  const parseValue = str => {
    const value = decodeURIComponent(str)
```

This would fix the tests, but I'm not sure why only the first value is decoded when there are more than one.

Reply:
Nevermind, the `*` in the pattern was causing this...